tests/dead_test.py

```diff
@@ -140,3 +140,16 @@ def test_partially_disabled(git_dir, capsys):
     assert dead.main(())
     out, _ = capsys.readouterr()
     assert out == 'x is never read, defined in f.py:1, f.py:3\n'
+
+
+def test_unused_argument(git_dir, capsys):
+    git_dir.join('f.py').write('def f(a, *b, c, **d): pass\nf')
```

Comment:
It might be good also to test that an argument appearing in the function body will cause that argument not to be included in the output.